ds-controller-board/res/configuration.toml

```diff
@@ -85,4 +78,12 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
   Retained = "false"
   AutoReconnect = "true"
   ConnectTimeout = "5" # Seconds
-  SkipCertVerify = "false"
\ No newline at end of file
+  SkipCertVerify = "false"
+
+[AppCustom]
+  [DriverConfig]
+  VID = '2341'
+  PID = '8037'
```

Comment:
I think VID and PID should stay as strings correct. But, they really should be moved to the protocol section of the Device definition. But that can be a separate PR.